### PR TITLE
docs: TAS5825M porting information

### PR DESCRIPTION
**Pull request by crawfxrd**
Should I add this to [smart-amp](https://github.com/system76/smart-amp) instead?

### CODE REVIEW

File: docs/tas5825m.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- Saleae logic analyzer
+- Saleae Logic 2
+- Micro SMD grabber test clips
+
+Be sure to unplug the battery before starting.
+
+### Attaching test clips
+
+It may not be feasible to probe the TAS5825M chip itself. Instead look
+for the MOSFET that connects to `SCL_5825` and `SDA_5825`, which will
```
Comment (jackpot51) [inline docs/tas5825m.md:21]: These lines often have different names. I'd just say "the smart amp's I2C clock and data lines"

File: docs/tas5825m.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+For example:
+
+- oryp5: Q55, located next to the SATA connector
+- oryp6: Q54, located above the M.2 drive slot
+
+In both these examples, pin 1 connects to the clock line and pin 4 connects to
+the data line. Attach a test clip to each pin, ensuring that they do not
+contact any other pins.
+
+Perform a literal smoke test. Plug in the AC adapter and power on the machine.
```
Comment (jackpot51) [inline docs/tas5825m.md:34]: I would add to remove the battery, so removing the AC powers off instantly

File: docs/tas5825m.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+### Parsing the data
+
+Clone the [smart-amp](https://github.com/system76/smart-amp) repo. Copy the
+output from the Logic 2 terminal to a new file named `<board>.txt` in the repo.
+
+Strip all lines that do no write to address 0x4E. Then strip everything
+but the data bytes from each line. Finally, trim the trailing spaces.
+
+```
+sed -i '/0x4E/!d' <file>
```
Comment (jackpot51) [inline docs/tas5825m.md:59]: Might as well make this part of the smart-amp repo, in a script

**Review by leviport — DISMISSED** [review]
I'll let Jeremy verify all of the technical information, but spelling, grammar, and structure all look good to me.

**Comment by crawfxrd** [comment]
Doc has been moved to system76/smart-amp#1. This now just updates the porting guide to point to it.

**Review by jackpot51 — APPROVED** [review]

**Review by leviport — APPROVED** [review]

**Pull request merged**